dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/FeatureManager.java

```diff
@@ -2491,9 +2494,9 @@ public void refreshFeatures(Filter filter) {
         refreshFeatures();
     }
 
-    boolean missingRequiredJava(FeatureResource fr) {
-        Integer requiredJava = fr.getRequireJava();
-        return requiredJava == null ? false : JavaInfo.majorVersion() < requiredJava;
+    public boolean withinJavaRange(FeatureResource fr) {
```

Comment:
Any reason you made this public?  It was default protected in the past.
```suggestion
    boolean withinJavaRange(FeatureResource fr) {
```